pylintrc

```diff
@@ -53,7 +53,8 @@ disable=
     too-many-return-statements,
     too-many-branches,
     too-many-arguments,
-    too-many-locals
+    too-many-locals,
+    missing-docstring
```

Comment:
But then no one ever has to write docstrings again.  Even a diligent developer who looks at the pylint errors won't be told that they are missing a docstring.